chapter_convolutional-neural-networks/pooling_vn.md

```diff
@@ -64,7 +64,11 @@ Instead, pooling operators are deterministic, typically calculating either the m
 These operations are called *maximum pooling* (*max pooling* for short) and *average pooling*, respectively.
 -->
 
-*dịch đoạn phía trên*
+Giống như ở các tầng tích chập, các toán tử gộp bao gồm một cửa sổ có kích thước cố định thực hiện trượt trên tất cả các vùng ở đầu vào dựa trên sải bước của nó,
+thực hiện xuất một giá trị cho mỗi vị trí mà cửa sổ của nó duyệt qua (đôi lúc được gọi là *cửa sổ gộp*).
+Tuy nhiên, không giống như các phép toán tương quan chéo giữa các đầu vào và các bộ lọc ở tầng tích chập, tầng gộp không chứa bất kỳ tham số nào (ở đây không có "bộ lọc").
+Thay vì vậy, các toán tử gộp là xác định với phép tính thường thực hiện là lấy giá trị cực đại hoặc trung bình của các phần tử trong cửa sổ thực hiện gộp.
```

Comment:
```suggestion
Thay vì vậy, các toán tử gộp là tất định với phép tính thường thực hiện là lấy giá trị cực đại hoặc trung bình của các phần tử trong cửa sổ thực hiện gộp.
```

Reply:
Chữ tất định này mình ít thấy người dùng và có đọc qua.

Reply:
mình thì hiểu `xác định` kiểu `defined` thôi chứ nó ko có nghĩa `deterministic` như `tất định`.

Reply:
defined thì mình sẽ dịch là định nghĩa còn chữ tất định thì nghe nó hơi khó hiểu và có lẽ hơi bí ẩn với nhiều người thích kiểu ngôn ngữ bình dân .

Reply:
https://www.rung.vn/dict/en_vn/Deterministic mình thì đọc trong các tài liệu toán tin trước giờ đều thấy xài vậy thôi nhé. cc @cuongvng . Tuỳ mọi người.